src/GovUk.Education.ExploreEducationStatistics.Publisher/Services/Interfaces/IPublishingCompletionService.cs

```diff
@@ -0,0 +1,13 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using GovUk.Education.ExploreEducationStatistics.Publisher.Model;
+
+namespace GovUk.Education.ExploreEducationStatistics.Publisher.Services.Interfaces;
+
+public interface IPublishingCompletionService
+{
+    Task CompletePublishingIfAllPriorStagesComplete(IEnumerable<(Guid ReleaseId, Guid ReleaseStatusId)> releaseAndReleaseStatusIds, DateTime publishedDate);
```

Comment:
Check formatting here as it looks like this long line should be wrapped.

Reply:
Sorted ta